src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -98,13 +95,14 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> sendAssignToJudgeCon
 
         paperNotificationService.printAssignToJudgeNotification(caseDetails, authToken);
 
-        if (!caseDataService.isPaperApplication(caseData) && caseDataService.isApplicantSolicitorAgreeToReceiveEmails(caseDetails)) {
+        if (!caseDataService.isPaperApplication(caseData)
+            && caseDataService.isApplicantSolicitorAgreeToReceiveEmails(caseDetails)) {
             log.info("Sending email notification to Applicant Solicitor for Judge successfully assigned to case");
             notificationService.sendAssignToJudgeConfirmationEmailToApplicantSolicitor(caseDetails);
         }
 
         if (featureToggleService.isRespondentJourneyEnabled()
```

Comment:
Can we look at removing the check for isRespondentJourneyEnabled as this feature has been in for a while now and is likely never to be switched off?